openwrt-openwisp-monitoring/files/monitoring.config

```diff
@@ -0,0 +1,3 @@
+config monitoring 'monitoring'
+	option monitored_interfaces '*'
+	option interval '300'
```

Comment:
If we use a [`time_to_seconds`](https://github.com/openwrt/packages/blob/791892d482c2255c8681b51bb0b78aa06d48f0e0/utils/watchcat/files/watchcat.init#L19) helper function then the user will be able to enter various strings like: `1d` for 1 day, `10h` for 10 hours, `5m` for 5 minutes, `10` or `10s` for 10 seconds

Reply:
What if user adds time like 10h30m? Should that case be handled too?

Reply:
@nickberry17 I don't see that, since we then have to parse that, but I don't care. You can already do it the way you suggested.

Reply:
> What if user adds time like 10h30m? Should that case be handled too?

I use `time_to_seconds` helper function mentioned above for now. We can open an issue for discussion if needed.

Reply:
>What if user adds time like 10h30m? Should that case be handled too?


I don't think it's necessary to cover every case